backend/src/campaignAnalysis.ts

```diff
@@ -30,99 +27,40 @@ export function getDateWithHighestAllConversions(rawSingleCampaignsData: RawSing
     date: string
     all_conversions: number
 } {
-    const sorted = rawSingleCampaignsData.sort((c1, c2) => c2.metrics.all_conversions - c1.metrics.all_conversions);
-    return { date: sorted[0].segments.date, all_conversions: Number(sorted[0].metrics.all_conversions.toFixed())}
+    return {
+        date: 'yyyy-mm-dd',
+        all_conversions: 0,
+    }
 }
 
 /**
  *
  * @returns a sum of all the clicks throughout the data
  */
 export function getTotalClicks(rawSingleCampaignsData: RawSingleCampaignsData[]): number {
-    let totalClicks = 0;
-    rawSingleCampaignsData.forEach(c => { totalClicks += c.metrics.clicks });
-    
-    return totalClicks;
+    return 0
 }
 
 /**
  *
  * @returns a sum of all the conversions throughout the data
  */
 export function getTotalAllConversions(rawSingleCampaignsData: RawSingleCampaignsData[]): number {
-    let totalAllConversions = 0;
-    rawSingleCampaignsData.forEach(c => { totalAllConversions += Number(c.metrics.all_conversions.toFixed()) });
-
-    return totalAllConversions;
+    return 0
 }
 
 /**
  *
- * HELPER FUNCTION
- * @returns an array of missing dates in an array of date strings
+ * @returns an array of clicks to be used in the chart
  */
-export function getMissingDates(dateArray: string[]){
-    try {
-        let result: string[] = []
-        for (let i = 0; i < dateArray.length; i++) {
-            let date1 = new Date(dateArray[i])
-            let date2 = new Date(dateArray[i+1])
-            let diff = date2.getDate() - date1.getDate()
-            if (diff > 1) {
-                while (diff > 1) {
-                    const newDate = new Date(date1.setDate(date1.getDate()+(diff-1))).toISOString().slice(0,10);
-                    result.push(newDate);
-                    diff--;
-                }
-            }
-        }
-        return result
-    } catch (e) {
-        console.log(e)
```

Comment:
Shouldn't the user of this function catch this exception and do something with it?